public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getStyles = ({ v1: { colors: v1Colors } }: GrafanaTheme2) => ({
```

Comment:
You can access the blue link color from `theme.colors.text.link` in `GrafanaTheme2`.

<img width="397" alt="image" src="https://user-images.githubusercontent.com/32638572/223065638-654aef98-b583-4ece-aafb-d3605b6079cb.png">

Don't know if that's the same as the one you need, so feel free to ignore this if it isn't.

Reply:
Yes it is same, changed it.